projects/swimlane/ngx-ui/src/lib/components/radiobutton/radiobutton.component.ts

```diff
@@ -58,14 +58,15 @@ export class RadioButtonComponent implements ControlValueAccessor {
       this.onChangeCallback(this._value);
     }
 
-    this.cdr.markForCheck();
+    this.cdr.detectChanges(); // Update this component
+    this.cdr.markForCheck(); // Update the host component (radio group)
   }
 
   @Input()
-  get value() {
+  get value(): any {
     return this._value;
   }
-  set value(value: boolean) {
+  set value(value: any) {
```

Comment:
Is this change of the type to support passing 'true' and 'false' as strings?

Reply:
This component has some weirdness... it confuses `value` and `checked`.  In our usage `value` is always a string.